tools/gen_objcdoc.sh

```diff
@@ -17,6 +17,13 @@ if [ $# -eq 0 ]
       exit 1
 fi
 
+# If GEM_HOME is set, prefer using its copy of jazzy.
+# LUCI will put jazzy here instead of on the path.
+if ! [ -z "${GEM_HOME}" ]
```

Comment:
Unless there's a semantic difference between `! [ expr ]` and `[ ! expr ]` that I'm unaware of in bash (which is entirely possible), use:
```
if [ ! -z "${GEM_HOME} ]
```
for consistency with the check on line 8.